rts/lra-examples/cdi-participant-with-coordinator/pom.xml

```diff
@@ -33,30 +34,48 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+<!--
+        <repository>
+            <id>jboss-public-releases-repository-group</id>
+            <name>JBoss Public Releases Maven Repository Group</name>
+            <url>https://repository.jboss.org/nexus/content/repositories/releases</url>
+        </repository>
+-->
+        <repository>
+            <id>repo.eclipse.org-snapshots</id>
+            <name>Eclipse MicroProfile Repository - Snapshots</name>
+            <url>https://repo.eclipse.org/content/repositories/microprofile-snapshots</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
```

Comment:
this can probably also be removed now as quickstarts depend on 1.0-RC1? Or do we plan to keep them up to date until final release?

Reply:
We will still need the snapshots as soon as we push changes in preparation for 1.0-RC2 or 1.0-Final